src/components/ProviderLoginButton/ProviderLoginButton.tsx

```diff
@@ -23,9 +26,10 @@ const ProviderLoginButton: React.FC<ProviderLoginButtonsProps> = (props) => {
                 onError: (err: any) => {
                     console.log(err)
                 },
-                // onSuccess: () => {
-                //     console.log("login successful")
-                // }
+                onSuccess: () => {
+                    navigate('/dashboard/home')
+                    onClose()
+                }
```

Comment:
I believe if mutateAsync is used here, it could fix the change of the AppBar button to wait on finding the user in the database first